taqueria-sdk/index.ts

```diff
@@ -387,14 +387,20 @@ export const getSandboxAccountConfig = (parsedArgs: RequestArgs.t) =>
 /**
  * Gets the initial storage for the contract
  */
-export const getInitialStorage = (parsedArgs: RequestArgs.t) =>
-	(contractFilename: string) => {
-		const env = getCurrentEnvironmentConfig(parsedArgs);
-
-		return env
-			? env.storage && env.storage[contractFilename]
-			: undefined;
-	};
+export const getInitialStorage = async (parsedArgs: RequestArgs.t, contractFilename: string) => {
+	const env = getCurrentEnvironmentConfig(parsedArgs);
+	if (env && env.storage && env.storage[contractFilename]) {
```

Comment:
Not a big deal, just coding style preference:

if we instead do:

```ts
if (!env || !env.storage || !env.storage[contractFilename]) {
  return undefined;
}
```
Then we can reduce the nesting.
Furthermore, we can simplify it to: `if (!env?.storage?.[contractFilename])`

Reply:
hmm I'd usually agree but the nesting is so shallow in this case 😅